suite-native/module-settings/src/screens/SettingsPrivacyAndSecurity.tsx

```diff
@@ -48,15 +48,14 @@ const DiscreetModeSwitchRow = () => {
 };
 
 const AnalyticsSwitchRow = () => {
-    const dispatch = useDispatch();
     const isAnalyticsEnabled = useSelector(selectIsAnalyticsEnabled);
 
     const handleAnalyticsChange = (isEnabled: boolean) => {
         if (isEnabled) {
-            dispatch(enableAnalyticsThunk());
+            analytics.enable();
             return;
         }
-        dispatch(disableAnalyticsThunk());
+        analytics.disable();
```

Comment:
Same here. You will turn it off but we need to also send this information somewhere